src/components/chartRepo/ChartRepo.tsx

```diff
@@ -261,33 +314,106 @@ function ChartForm({ id = null, name = "", active = false, url = "", authMode =
+                    <>
+                        <CustomInput
+                            autoComplete="off"
+                            value={customState.username.value}
+                            onChange={customHandleChange}
+                            name="username"
+                            error={customState.username.error}
+                            label="Username*"
+                            labelClassName="mt-12"
+                        />
+                        <ProtectedInput
+                            value={customState.password.value}
+                            onChange={customHandleChange}
+                            name="password"
+                            error={customState.password.error}
+                            label="Password*"
+                            labelClassName="mt-12"
+                        />
+                    </>
+                )}
             </div>
             {/* <div className="form__label">Authentication type*</div>
             <div className="form__row form__row--auth-type pl-12 pointer">
-                {[{ label: 'User auth', value: 'USERNAME_PASSWORD' }, { label: 'Password/Auth token', value: "ACCESS_TOKEN" }, { label: 'Anonymous', value: 'ANONYMOUS' },]
+                {[{ label: 'User auth', value: CHART_REPO_AUTH_TYPE.USERNAME_PASSWORD }, { label: 'Password/Auth token', value: "ACCESS_TOKEN" }, { label: 'Anonymous', value: 'ANONYMOUS' },]
                     .map(({ label: Lable, value }) => <label key={value} className="flex left pointer">
                         <input type="radio" name="auth" value={value} onChange={handleOnChange} checked={value === state.auth.value} /> {Lable}
                     </label>)}
             </div>
             {state.auth.error && <div className="form__error">{state.auth.error}</div>}
-            {state.auth.value === 'USERNAME_PASSWORD' && <div className="form__row form__row--two-third">
+            {state.auth.value === CHART_REPO_AUTH_TYPE.USERNAME_PASSWORD && <div className="form__row form__row--two-third">
                 <CustomInput value={customState.username.value} onChange={customHandleChange} name="username" error={customState.username.error} label="Username*" />
                 <ProtectedInput value={customState.password.value} onChange={customHandleChange} name="password" error={customState.password.error} label="Password*" />
             </div>}
             {state.auth.value === "ACCESS_TOKEN" && <div className="form__row">
                 <ProtectedInput value={customState.accessToken.value} onChange={customHandleChange} name="accessToken" error={customState.accessToken.error} label="Access token*" />
             </div>} */}
+            {chartRepoType !== CHART_REPO_TYPE.PUBLIC && (
+                <Checkbox
+                    rootClassName="fs-13 dc__hover-n50 pt-8 pb-8 pl-8 ml-8"
+                    isChecked={secureWithTls === true}
```

Comment:
we can directly use isChecked={secureWithTls}

Reply:
done